examples/mock/README.md

```diff
@@ -0,0 +1,25 @@
+# GovStack Consent BB - mock
```

Comment:
Can we get some level of “genericisation” here? Make it clear that this is a template mock api rather than one for consent?